src/main/java/org/isf/menu/gui/MainMenu.java

```diff
@@ -326,6 +345,35 @@ public void windowClosing(WindowEvent e) {
 		setVisible(true);
 	}
 
+	private HashMap<String, Boolean> retrieveActivatedModulesMap() {
+		return new HashMap<String, Boolean>() {
```

Comment:
No need to define the map's types; it can determine it from the method's signature

Reply:
done, thanks!